app/i18n.js

```diff
@@ -0,0 +1,20 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+const languages = {
+    en: { translation: require("./lang/en.json") }
```

Comment:
don't use double quotes in require

Reply:
@saghul you should add a check on travis CI running `npm run lint` at least

Reply:
Looks like the PR does not lint also:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ app/features/onboarding/constants.js:39:17

Missing type annotation for props.

     36│         dialogPlacement: 'top right',
     37│         target: 'settings-drawer-button',
     38│         text: 'onboarding.settingsDrawerButton',
     39│         onNext: props => props.dispatch(openDrawer(SettingsDrawer))
     40│     },
     41│     'name-setting': {
     42│         type: OnboardingSpotlight,


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ app/features/onboarding/constants.js:76:17

Missing type annotation for props.

     73│         dialogPlacement: 'top right',
     74│         target: 'always-on-top-window',
     75│         text: 'onboarding.alwaysOnTop',
     76│         onNext: props => setTimeout(() => {
     77│             props.dispatch(closeDrawer());
     78│         }, 300)
     79│     }


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ app/features/settings/components/SettingToggle.js:24:25

Cannot apply type because it is not a polymorphic type.

     21│     /**
     22│      * A function to produce setting change events.
     23│      */
     24│     settingChangeEvent: Function<Boolean, Object>;
     25│
     26│     /**
     27│      * Redux dispatch.
```